packages/grid/src/features/standard-table/hooks/UseStandardTableConfig.ts

```diff
@@ -0,0 +1,43 @@
+import { useContext } from "react";
+import { StandardTableConfig } from "../config/StandardTableConfig";
+import {
+  StandardTableActionsContext,
+  StandardTableConfigContext,
+  StandardTableInternalActionsContext,
+  StandardTableStateContext,
+  StandardTableTableIdContext
+} from "../context/StandardTableStateContext";
+import { StandardTableState } from "../redux/StandardTableReducer";
+
+export const useStandardTableId = (): string => {
+  return useContext(StandardTableTableIdContext) as string;
```

Comment:
why define the type as `string | undefined` and then just cast it to `string`? either keep it type-safe or not :D

Reply:
Default value is undefined. Maybe there is a better solution?